bin/eshost.js

```diff
@@ -193,11 +195,19 @@ if (argv.add) {
   hostManager.add(config, argv.add[0], argv.add[1], argv._[0], argv.args, hostTags);
   console.log(`Host '${argv.add[0]}' added`);
   process.exit(0);
-} else {
-  if (argv.args) {
-    console.error('Use --args with --add');
-    process.exit(1);
-  }
+}
+
+// edit a host
+if (argv.edit) {
+  hostManager.edit(config, argv.edit, argv.args, hostTags);
+  console.log(`Host '${argv.edit}' edited`);
+  process.exit(0);
+}
+
+if (argv.args) {
+  // at this point in execution, implies (argv.args && !(argv.add || argv.edit))
```

Comment:
I think this would be more clear as a check on line 190 with the contents of this comment.